ryanoasis/nerd-fonts
# Add the official Puppet icon as a custom icon

#### Description

Added the official icon for the Puppet DSL to the nerd fonts.

#### Requirements / Checklist

  - If any of the scripts were modified they have been tested and execute without error, e.g.:
    - `./font-patcher Inconsolata.otf --fontawesome --octicons --pomicons`
    - `./gotta-patch-em-all-font-patcher\!.sh Hermit`

#### What does this Pull Request (PR) do?

* Add the Puppet SVG (https://logos-download.com/wp-content/uploads/2019/01/Puppet_Labs_Logo.png)
* Modify the original-source.otf font with fontforge (e531 is Puppet, OpenType CFF)
* Add it to the font-patcher


#### How should this be manually tested?

* Patch a font of your choice with font-patcher. Be sure to include the Seti/Custom icons
* Install the font to your system
* Open the `bin/scripts/lib/l_seti.sh` file and make sure the icon appears correctly at the `i_custom_puppet` location

#### Any background context you can provide?

Many of the icons out there, such as devicons and nvim-webdev-icons still utilize the a beaker as the icon for Puppet code manifests, which was changed several years ago to the current icon.

#### What are the relevant tickets (if any)?

#### Screenshots (if appropriate or helpful)

![Screenshot from 2021-08-13 12-59-23](https://user-images.githubusercontent.com/2914211/129400429-a364aa34-429a-4415-a967-819efea57338.png)
![Screenshot from 2021-08-13 12-59-43](https://user-images.githubusercontent.com/2914211/129400432-1ccdf0a5-8e4a-4c13-b34c-651bd00e83e3.png)
